scripts/gen_handles.py

```diff
@@ -62,6 +62,8 @@ def parse_args():
 
     parser.add_argument("-k", "--kernel", required=True,
                         help="Input zephyr ELF binary")
+    parser.add_argument("-d", "--dynamic-devices", required=True,
+                        help="Input number of dynamic devices allowed")
```

Comment:
why does this need to be a required argument ?

Why not make it optional, and if not provided treat it as `0`.